### ISSUE TITLE
onupdate failing

### ISSUE THREAD

**Issue by jondashkyle** [yoshuawuyts/nanocomponent#7]
Getting the error `Uncaught TypeError: CreateListFromArrayLike called on non-object` (line 119) when `onupdate` is defined. Looking into it, but thought to drop a quick note here!

```js
component({
  onupdate: function (el, data) {
    console.log(el, data)
  },
  render: function (data) {
    return h`<div>hello!</div>`
  }
})
```

Should mention this is in use w/ choo, and am running into the same issue with the example.

**Comment by alterx** [comment]
I saw this earlier today, turns out it's related to a bug that's addressed in this PR 
https://github.com/yoshuawuyts/nanocomponent/pull/8

**Comment by jondashkyle** [comment]
Ah! Good catch.

**Comment by alterx** [comment]
Hey @jondashkyle , idk if we want to close this one before the PR gets merged. Maybe just keep it open and then close when @yoshuawuyts reviews #8 ?

**Comment by yoshuawuyts** [comment]
Sorry for keeping this open for so long; another project took all my attention for a lil while - published in `v2.0.1` :sparkles:

@jondashkyle added you as a maintainer; I kinda feel bad this PR was held up for so long and think it's a good idea if other people are able to review & merge if anything like this comes up again. Cheers!
